lms/static/js/student_account/views/LoginView.js

```diff
@@ -126,7 +126,9 @@
                                 paragraphEnd: HtmlUtils.HTML('</p>'),
                                 email: email,
                                 platform_name: this.platformName,
-                                anchorStart: HtmlUtils.HTML('<a href="' + this.supportURL + '">'),
+                                anchorStart: HtmlUtils.HTML(
+                                    '<a target="_blank" href="' + this.passwordResetSupportURL + '">'
```

Comment:
Was there a requirement to include `target="_blank"`? It's generally a worse user experience to force new windows/tabs on the user - better to let them decide. So I'd rather leave this as it was if there's no request to change it.